packages/frontend/src/pages/invitation/invitation.page.tsx

```diff
@@ -37,42 +58,45 @@ export const InvitationPage: React.FC = () => {
     searchParams.get('spaceOwnerEmail') || '',
   );
 
+  const { mutate: mutateAcceptInviteHandle } = useMutation({
+    mutationFn: ({ token, spaceOwnerEmail }: IAccepteInvite) =>
+      fetchAcceptInviteHandle({ token, spaceOwnerEmail }),
+    onError: (error) => {
+      const err = error as unknown as AxiosError<IErrorResponse>;
+      console.log(err);
+    },
+    onSuccess: () => {
+      navigate('/');
+    },
+  });
+  const { data: userData, isLoading } = useQuery(['user-inviter'], () =>
+    fetchUser(spaceOwnerEmail, token ?? ''),
+  );
+
   useEffect(() => {
     if (!token) {
       return;
     }
-    fetchUser(spaceOwnerEmail, token).then((result) => {
-      if (result) {
-        setResponse(result);
-        setChangeDefaultUserSpace(result.email);
-      }
+    if (isLoading) {
+      setLoading(true);
```

Comment:
Please get rid of own component loading state

Reply:
Done